src/components/Nav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+import '../styles/nav.css';
+
+const Nav = () => (
+  <nav>
+    <h1>Bookstore CMS</h1>
+    <ul className={Nav.ul}>
+      <li id="nav-book"><Link to="/">BOOKS</Link></li>
+      <li id="nav-categories"><Link to="/categories">CATEGORIES</Link></li>
+    </ul>
+    <div id="user-icon">
+      <FontAwesomeIcon icon={faUser} style={{ color: '#0290ff' }} />
+    </div>
+  </nav>
```

Comment:
- [ ] Hi @HossainAraf, you did a great job by implementing the project but it seems like while implementing the `navigation` you missed the `user` icon as per the `Zeplin's` design. Kindly follow the requirement properly because your design should match **exactly** with the *Zeplin* CMS design.

### Missing (Look at the right hand side of the below screenshot)

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/494bb09f-2e30-4399-9246-2e224a3354a4)

Reply:
Hi @skyv26 . I'm confused. Because I can see the the user Icon in my machine. Will you please recheck it?

Reply:
@HossainAraf I can see your design is overflowing and that's the reason why I was not able to see your icon. Kindly make sure that your design is not overflowing.

![image](https://github.com/HossainAraf/book-store-spa/assets/66476329/0b07e96f-3356-4908-b266-696c82a92baf)

Reply:
Ok, thanks